### ISSUE TITLE
how to pretrain model on MS-COCO (Deeplabv3+) 

### ISSUE THREAD

**Issue by yswang1717** [tensorflow/models#5008]
I'm training deeplabv3+ with my own backbone network and I want to pretrain my models on MS-COCO dataset.  the deeplab paper shows that pre-train models on coco dataset can much improve the results, but the coco dataset just give the instance level labels, how to transfer data to satisfy the category level requirement, or anyone can give me some examples on how to pre-train models on coco datasets? and Do you have any idea how to handle overlapping segments in the MS-COCO ground truth while training the network?

Thank you for your quick reponse. The followings are details of my version.
What is the top-level directory of the model you are using
/home/semantic_segmentation/models/research/deeplab/
Have I written custom code
No
OS Platform and Distribution
Ubuntu 16.04.3 LTS ( GNU/Linux 4.4.0-98-generic x86_64)
TensorFlow installed from
pip command
TensorFlow version
1.6.0
Bazel version
None
CUDA/cuDNN version
None
GPU model and memory
TITAN Xp 12GB * 8
Exact command to reproduce
None.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by yswang0522** [comment]
Thank you for your quick reponse. The followings are details of version. 
What is the top-level directory of the model you are using 
/home/semantic_segmentation/models/research/deeplab/ 
Have I written custom code
No
OS Platform and Distribution
Ubuntu 16.04.3 LTS ( GNU/Linux 4.4.0-98-generic x86_64) 
TensorFlow installed from
pip command
TensorFlow version
1.6.0 
Bazel version
None
CUDA/cuDNN version
None 
GPU model and memory
TITAN Xp 12GB * 8 
Exact command to reproduce
None.

**Comment by tt-leader** [comment]
Maybe the paper [Weakly- and Semi-Supervised Learning of a DCNN for Semantic Image Segmentation](http://arxiv.org/abs/1502.02734) could help you.

**Comment by Damanpreet** [comment]
hi @yswang0522, Were you able to train Deeplab on COCO? What training configuration did you use?

**Comment by kumariko** [comment]
@yswang1717 We are checking to see if you still need help on this issue? Could you please have a look on the [link1](https://github.com/tensorflow/models/issues/3643), [link2](https://stackoverflow.com/questions/50619150/how-can-i-load-a-pre-trained-model-on-coco-dateset-for-image-segmentation) and let us know if it helps? Thanks!
